cmd/mapstructure-to-hcl2/mapstructure-to-hcl2.go

```diff
@@ -201,6 +201,7 @@ type StructDef struct {
 	Struct             *types.Struct
 }
 
+// outputStructHCL2SpecBody writes the hcl spec of a Config
```

Comment:
```
// outputStructHCL2SpecBody writes the map[string]hcldec.Spec that defines the HCL spec of a 
// struct. Based on the layout of said struct.
// If a field of s is a struct then the HCL2Spec() function of that struct will be called, otherwise a 
// cty.Type is outputed.